test/gui/shared/scripts/pageObjects/AccountConnectionWizard.py

```diff
@@ -183,13 +183,20 @@ def basicLogin(username, password):
 
     @staticmethod
     def oidcLogin(username, password):
+        AccountConnectionWizard.browser_login(username, password, 'oidc')
+
+    @staticmethod
+    def oauthLogin(username, password):
+        AccountConnectionWizard.browser_login(username, password, 'oauth')
+
+    @staticmethod
+    def browser_login(username, password, login_type=None):
```

Comment:
I thought we are using :camel: - case function names
```suggestion
    def browserLogin(username, password, login_type=None):
```
same for `authorize_via_webui`, but that can be fixed in a separate PR